2021/warmerdam2022re/openreview/content.tex

```diff
@@ -0,0 +1,584 @@
+% (Note: you'll likely have to record this as you run your experiments, so it's better to think about it ahead of time). Generally, consider the perspective of a reader who wants to use the approach described in the paper --- list what they would find useful.
+We had access to a single GeForce 1080 Ti GPU from the Lisa Cluster
+\cite{lisac}, which has a Bronze 3104 (1.7GHz) processor with  12 CPU Cores and 256GM RAM of memory. % and 4x GeForce 1080Ti, 11 GB GDDR5x GPUs.
+
+For the Cifar100 dataset with the ResNet architecture, a complete trainingcycle took 2 hours and evaluating it under the reconstruction attack took 10 hours in order to reconstruct 100 images. The policy search took about 1 minute per policy.
+
+
+\section{Results}
+
+\subsection{Reproduction}
+
+\textbf{Correlation privacy score and reconstruction score.} In the original paper the authors show that their proposed privacy score has a positive correlation with the reconstruction score of the attacker. A comparison between their results and the reproduced results can be found in Figure \ref{fig:correlation}. It can be seen that the results in the figures do not match. When fitting a linear trend between the metrics, there appears to be no correlation in the reproduced results. Possible explanations for this are explained in Section \ref{sec:discuss}.
+
+\begin{figure}[htp]
+    \centering
+    \includegraphics[width=12cm]{pics/correlation_comparison.png}
+    \caption{Empirical validation of the correlation between the reconstruction (PSNR) score and the privacy score}
+    \label{fig:correlation}
+\end{figure}
+
+
+\textbf{The Performance of Selected Policies} \citeauthor{gao2021privacy} claim the policies selected by the search algorithm reduce the PSNR reconstruction score significantly while not resulting in a great loss of accuracy. The results presented in the paper along with the results that resulted from the reproduction can be seen in Table \ref{tab:resls}.
+
+
+Some results from the reproduction differ substantially from the ones presented in the paper, as can be seen from the red entries in Table \ref{tab:resls}. The biggest difference is the reconstruction score for the unaugmented policy of the Cifar100. Which differs greatly from the reconstruction score presented in the original paper. However, it is still significantly higher than that of the augmented policies.
+
+The model's accuracies are comparable in most instances. But, the accuracy found in this research on the \textit{3-1-7} policy of the Cifar100 dataset scores almost 13\% below the accuracy of the unaugmented policy, compared to 6\% in the original paper.
+
+
+% \begin{table}[h]\label{tab:resls}
+% \begin{subtable}
+% \begin{tabular}{l|l l l l}
+%       \textbf{Policy} &  \textbf{Theirs} & \textbf{Ours} \\\hline
+%   None &  &  &   \\
+%   3-1-7 &  &  &  \\
+%   43-18-18 &  &  & \\
+%   Hybrid &  &  & \\
+%   \hline
+% \end{tabular}
+% \caption{x}
+% \end{subtable}
+\\
```

Comment:
I think the error is happening due to this linebreak, check https://www.overleaf.com/learn/latex/Errors/LaTeX_Error%3A_There%27s_no_line_here_to_end. Try commenting this out and see if it works!